api/client/stack/remove.go

```diff
@@ -63,6 +63,11 @@ func runRemove(dockerCli *client.DockerCli, opts removeOptions) error {
 		}
 	}
 
+	if len(services) == 0 && len(networks) == 0 {
```

Comment:
Yeah. If a stack was brought up with both networks and services, then the services were all manually deleted, you'd expect `stack rm` to still remove the networks.